packages/web/src/storage/cookie-store.ts

```diff
@@ -0,0 +1,92 @@
+import { SESSION_INACTIVITY_TIMEOUT_SECONDS } from "../session/constants";
+import { isIframe } from "../utils";
+import { throttle } from "../utils/throttle";
+
+const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent)
+
+export class CookieStore<T> implements Store<T> {
+	private _cachedValue: T | undefined;
```

Comment:
We did not enforce it by lint, but we try not to use `_` prefixes. The private keyword is enough. I know this is common in other languages, but if we want to follow JavaScript conventions, we should be prefixing it with #, which looks ugly and does not make sense. Hence, as a team, we decided to just use the private keyword and avoid prefixing.


https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties